src/components/v5/common/MemberSignatureList/MemberSignatureList.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC } from 'react';
+
+import { notNull, notUndefined } from '~utils/arrays';
+import { SpinnerLoader } from '~shared/Preloaders';
+import MemberSignature from '../MemberSignature';
+import { MemberSignatureListProps } from './types';
+
+const displayName = 'v5.common.MemberSignatureList';
+
+const MemberSignatureList: FC<MemberSignatureListProps> = ({
+  items,
+  isLoading,
+  title,
+  checkedUsersList,
+}) => {
+  // @TODO: Use this data in recovery mode signatures list component
```

Comment:
Please remove redundant comments. Hook with the logic that you've added shouldn't be used inside components directory, it should be used in a place where you'll use it.